app/views/audits/index.html.erb

```diff
@@ -2,7 +2,7 @@
   Audit log - <%= t('page_title') %>
 <% end %>
 
-<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Home", root_path %>
```

Comment:
please add root_path(filter_options: PlanningApplication::FILTER_OPTIONS) 
to fix https://trello.com/c/4FoMO0ys/1554-filters-are-marked-as-selected-but-they-are-not please 😄